src/include/txn.i

```diff
@@ -933,6 +933,8 @@ __wt_txn_read(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_ITEM *key, uint
         return (0);
     }
 
+    update_list_stop_tw = WT_TIME_WINDOW_HAS_STOP(&cbt->upd_value->tw);
```

Comment:
It took some time for me to understand the flag usage. I feel adding a comment to explain it in the following checks would be good.

Reply:
Agreed - the variable name is misleading here. I read `tw` as being a time window, so it's odd to assign a bool to it.